sky/build/sdk_xcode_harness/Flutter/Flutter.xcconfig

```diff
@@ -0,0 +1 @@
+#include "Generated.xcconfig"
```

Comment:
The value is that it's checked into source control and references a file that's outside source control.  The Xcode project can reference Flutter.xcconfig without breaking even if Generated.xcconfig doesn't exist.